x/mint/types/params.go

```diff
@@ -43,11 +43,11 @@ func NewParams(
 func DefaultParams() Params {
 	return Params{
 		MintDenom:           sdk.DefaultBondDenom,
-		InflationRateChange: sdk.NewDecWithPrec(13, 2),
-		InflationMax:        sdk.NewDecWithPrec(20, 2),
-		InflationMin:        sdk.NewDecWithPrec(7, 2),
+		InflationRateChange: sdk.NewDecWithPrec(1, 2),
```

Comment:
[question] if inflation is fixed at 2% can this be

```suggestion
		InflationRateChange: sdk.NewDec(0),
```

Reply:
good point, idk this was set to 1. used this instead https://github.com/celestiaorg/cosmos-sdk/pull/285/commits/0e05cb2cb0174c05ea1a0ee6298d9fb5a2e627cc